web/src/components/views/Transform.vue

```diff
@@ -95,6 +121,9 @@ v-container(fill-height)
             v-radio(v-for="m in scaling_methods", :label="m.label",
                 :value="m.value", :key="`scale${m.value}`")
     v-layout(column).pa-2
+      v-card
+        vis-pca(:width="500" :height="400" :points="points")
```

Comment:
Should probably address window resize, probably in another PR.  Probably when we understand more about the layout.

Reply:
Good point. I opened #43 for this.